scripts/mkgrub.sh

```diff
@@ -72,7 +72,7 @@ for TARGET in $TARGETS;do
     GRUB_MODULES="${CD_MODULES} ${FS_MODULES} ${PXE_MODULES} ${CRYPTO_MODULES} ${MISC_MODULES} ${TARGET_EXTRA_MODULES}"
     MODULE_DIR="${GRUB2_MOD_DIR}/${TARGET_MOD_DIR}"
     set -x
-    grub2-mkimage -O "${TARGET}" -o "${BOOTLOADERS_DIR}/grub/${BINARY}" --prefix= "${GRUB_MODULES}"
+    grub2-mkimage -O "${TARGET}" -o "${BOOTLOADERS_DIR}/grub/${BINARY}" --prefix= ${GRUB_MODULES}
```

Comment:
I see. We actually want shell splitting at the point since we want to give grub a list of the modules.

I see two options to fix the linter here.

1. Add `# shellcheck disable=SC2086` in the line before the change.
2. Preferred: As suggested here https://stackoverflow.com/a/62638388/4730773 we can split explicit and then read it compliant to shellcheck. So that would become then:

```suggestion
    read -ra grub_module_list <<<"${GRUB_MODULES}"
    grub2-mkimage -O "${TARGET}" -o "${BOOTLOADERS_DIR}/grub/${BINARY}" --prefix= "${grub_module_list[@]}"
```

Reply:
The 2nd becomes unnecessary complicated. Let's go for the first...